README.md

```diff
@@ -9,3 +9,22 @@
 R API for Naomi app
 
 App to show district level estimates of HIV indicators
+
+## Running in docker
+
+Docker images are built on travis, if on master branch run via
+
+```
+docker run --rm -d -p 8888:8888 mrcide/hintr:latest
```

Comment:
Consider adding a `--name=hintr` here and a `docker rm -f hintr` at the bottom to clean up